### ISSUE TITLE
Drop zones are draggable by default

### ISSUE THREAD

**Issue by andrehaveman** [akserg/ng2-dnd#25]
* **I'm submitting a ...**
[*] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**
Bug

* **What is the current behavior?**
Drop zones are draggable by default because a draggable attribute is added to a drop zone.

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via https://plnkr.co or similar.
Add [dnd-droppable] to an element

* **What is the expected behavior?**
Drop zones should not be draggable.

* **What is the motivation / use case for changing the behavior?**
Drop zones should only allow dropping other draggables, not be draggable itself.

* **Please tell us about your environment:**

- Angular version: 2.0.0-rc.1
- Browser: all

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)
Now the AbstractComponent sets `this.dragEnabled = true;' on line 63. The draggable component also sets the dragEnabled property. I think the draggable component should be the only one that sets dragEnabled to true so line 63 of AbstractComponent can be removed.

**Comment by akserg** [comment]
I agree that the issue exists. I will investigate it in closest time.

**Comment by akserg** [comment]
Fixed in 1.8.2

I added emitters for **dragStart** and **dragEnd** to the **DraggableComponent** as well. Please create the new issue for this one if it still necessary:

> Perfect thanks, onDropSuccess would be good as well so when dropping on a draggable object within the drop-target the dragged object can get added to the drag target as well.
